pkg/io/csvio/common.go

```diff
@@ -65,3 +65,7 @@ func printProtocolName(protocol ir.Protocol) string {
 	}
 	return ""
 }
+
+func vpcFromScopedResource(resource ir.ID) ir.ID {
```

Comment:
There should be only one copy of this function in the `ir` package, next to `ScopingComponent()`. Do not duplicate it per writer.
Also, there are still many other places where it can be used